lua/telescope/builtin/__internal.lua

```diff
@@ -1089,8 +1089,9 @@ internal.registers = function(opts)
         results = registers_table,
         entry_maker = opts.entry_maker or make_entry.gen_from_registers(opts),
       },
-      -- use levenshtein as n-gram doesn't support <2 char matches
-      sorter = sorters.get_levenshtein_sorter(),
+			-- levenshtein doesn't seem to work with ordinal = register content
+			-- so changed to get_generic_fuzzy_sorter
+      sorter = sorters.get_generic_fuzzy_sorter(),
```

Comment:
would be better to do `conf.generic_sorter(opts)` we also dont really need that comment

Reply:
What does that do though? Seems to work find in both ways

Reply:
using the sorter configured by the user rather than an hardcoded one.

So people use fzf-native if they have that one installed, or fzy-native, or fzy-lua,, ....